### ISSUE TITLE
Disable logging by default 

### ISSUE THREAD

**Issue by ghost** [mikeobrien/node-windows-service-controller#2]
node-windows-service-controller produces a lot of console.logs. I would like to make console.logs opt-in (user provides a flag saying that she wants to print to stdout). 

Thoughts?

**Comment by mikeobrien** [comment]
Good idea, I'd be open to a PR that adds a "silent" option. Would like the
default to remain verbose tho (as it's already in use in a number of build
scripts where verbose is required).

**Comment by rithvikvibhu** [comment]
Is anyone working on it yet? I'd really like this to be added.
If @mikeobrien is still open to this, I can submit a PR.

**Comment by BartaG512** [comment]
this would be very useful
